packages/exo/docs/exo-taxonomy.md

```diff
@@ -1,28 +1,32 @@
-* "exo" means "outside". An exo object is reachable from outside the vat. Other vats can hold a reference (a capability) to an exo, and anyone with a reference to an exo can send it messages. Put another way, the exo objects are the objects in a vat that are outward facing, and so must be fully defensive against receiving bad messages.
-* Alludes to "ExoSkeleton", a protective outside layer that is an effective first defense against many threats. Likewise, an exo's interface guard is a great first layer of type-like (pattern-based) input validation. The programmers remaining burden to make the outward facing exo objects fully defensive thus becomes easier.
+An Exo is a Far object protected by an interface guard. We chose the term "exo" because
+* It means "outside". An exo object is reachable from outside the vat. Other vats can hold a reference (a capability) to an exo, and anyone with a reference to an exo can send it messages.
+* It alludes to "ExoSkeleton", a protective outside layer that is an effective first defense against many threats. Likewise, an exo's interface guard is a great first layer of type-like ([pattern](https://github.com/endojs/endo/tree/HEAD/packages/patterns)-based) input validation protecting against many kinds of bad messages. The programmer's remaining burden to make the exo objects fully defensive thus becomes easier.
 
-"Exo" forms a nice pairing with Endo, our distributed Hardened JavaScript platform.
+"Exo" also forms a nice pairing with [Endo](https://github.com/endojs/endo) itself.
 
 ## Make instance vs Define class vs Define class kit
 
-* ***`make*Exo`*** <br>
-Makes a new fresh Exo instance.
-* ***`define*ExoClass`*** <br>
-Each call defines a class-like category of Exo instances, where the arguments of the `define*ExoClass` call describe what all instances of that category have in common. Each call makes and returns a new fresh `make*` function that makes new instances of that category. The arguments of that returned `make*` function describe what is specific to the instance it makes.
-* ***`define*ExoClassKit`*** <br>
-We often call a record of named entangled Xs an "XKit", as it a "toolkit" is a collection of closely related tools. Each call to `define*ExoClassKit` defines a kit of entangled Exo classes. Each call makes and return a single new fresh `make*Kit` function for making a corresponding kit of Exo instances. Each of these instances is considered to be a "facet" of that kit. All facets of the same kit share a common encapsulated state.
+### make*Exo
+Each call to `makeExo` or `makeDurableExo` makes and returns a new fresh Exo instance.
+
+### define*ExoClass
+Each call to `defineExoClass`, `defineVirtualExoClass`, or `defineDurableExoClass` defines a class-like category of Exo instances and makes and returns a "make" function that makes new instances of that category. The arguments of that returned function describe what is specific to the instance it makes.
```

Comment:
```suggestion
Each call to `defineExoClass`, `defineVirtualExoClass`, or `defineDurableExoClass` defines a class-like category of Exo instances, and makes and returns a "make" function that makes new instances of that category. The arguments of that returned function describe what is specific to the instance it makes.
```

I still think it is worth also saying that the arguments to the `define` call describe what the instances of that category have in common. The juxtaposition helps.

Reply:
Restored the juxtaposition.